IconButton/IconButton.module.less

```diff
@@ -13,6 +13,7 @@
 		box-sizing: border-box;
 		font-size: @agate-iconbutton-font-size;
 		vertical-align: middle;
+		text-indent: 0;
```

Comment:
Let's make sure to not lose this with the deprecation of IconButton in #169. This is important for keeping the icon centered inside the Button.